Directory.Packages.props

```diff
@@ -1,12 +1,10 @@
 <Project>
-
```

Comment:
Revert the removal of these empty lines please. We intentionally put them between property / items groups and after the project start and before the project closing tag for readability.

Reply:
This is done by the linter I guess. I'll fix it.